cluster-autoscaler/core/static_autoscaler_test.go

```diff
@@ -45,17 +46,25 @@ import (
 	"k8s.io/klog"
 )
 
-type nodeListerMock struct {
-	mock.Mock
+type fakeNodeLister struct {
```

Comment:
Please renamte to `TestNodeLister` and move to `testlister.go` making it consistent with other test listers.
Please also add the `NewTestNodeLister(nodes...) TestNodeLister` method. And if you neet mutability `TestNodeLister.SetNodes()`

Reply:
Changed to be consistent but I think we use 'fake' prefix for fake implementations.